packages/contracts-bedrock/scripts/L2Genesis.s.sol

```diff
@@ -230,6 +232,7 @@ contract L2Genesis is Deployer {
         // 1B,1C,1D,1E,1F: not used.
         setSchemaRegistry(); // 20
         setEAS(); // 21
+        setSoulGasToken(); // FE
```

Comment:
The comment should be `// 800`?

Reply:
Fixed, I made a forced push to make the address change within a single commit.